zone/gm_commands/spell_max_thresholds.cpp

```diff
@@ -0,0 +1,178 @@
+#include "../command.h"
+
+void command_spell_max_thresholds(Client* c, const Seperator* sep)
+{
+	const int arguments = sep->argnum;
+	if (arguments) {
+		const bool is_help = !strcasecmp(sep->arg[1], "help");
+
+		if (is_help) {
+			std::vector<std::string> description =
+			{
+				"Threshold of your own health when bots will start casting the chosen spell type"
+			};
```

Comment:
Why is this command that appears to be bot specific not in the bot gm command section ?

Reply:
Move all these GM commands for clients to the bot commands and added client checks to them